app/helpers/message_thread_helper.rb

```diff
@@ -0,0 +1,9 @@
+module MessageThreadHelper
+  def self.show_recipient?(message_thread)
+    message_thread.box.single_recipient? && message_thread.is_outbox && message_thread.recipient.present?
```

Comment:
imho mas tu logiku naopak. ak je to single recipient tak nema zmysel ukazovat adresata.